src/Avalonia.Visuals/Animation/Transitions/BrushTransition.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using Avalonia.Animation.Animators;
+using Avalonia.Media;
+
+namespace Avalonia.Animation
+{
+    /// <summary>
+    /// Transition class that handles <see cref="AvaloniaProperty"/> with <see cref="IBrush"/> type.
+    /// Only values of <see cref="ISolidColorBrush"/> will correctly transition.
+    /// </summary>
+    public class ISolidColorBrushTransition : Transition<IBrush>
```

Comment:
I would prefer to go with SolidColorBrushTransition

Reply:
Should I rename the animator as well?

Reply:
Probably, but it will be a minor breaking change. On other hand I don't think anybody uses it directly.

Reply:
Actually we already had a `SolidColorBrushAnimator` which is deprecated now. I named it like this to keep it consistent between animators and transitions.

Reply:
I've renamed it to `BrushTransition` since it handles all brushes now (at least in theory).